admin_pages/registrations/Registrations_Admin_Page.core.php

```diff
@@ -1506,7 +1506,13 @@ protected function _get_orderby_for_registrations_query()
         if (empty($orderby['REG_ID'])) {
             $orderby['REG_ID'] = $order;
         }
-        return array('order_by' => $orderby);
+        return apply_filters(
```

Comment:
If this is intended to change the order_by value only, it'd be better to just filter the `$orderby` value and not the entire value.